pkg/boot/align/align.go

```diff
@@ -1,15 +1,23 @@
 // Copyright 2015-2022 the u-root Authors. All rights reserved
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
-
 package align
 
 import "os"
 
-var pageMask = uint(os.Getpagesize() - 1)
+var pageSize = uint(os.Getpagesize())
+
+func AlignUpBySize(v uint, alignSize uint) uint {
```

Comment:
Why does this align up to a page size if they ask for less than a page size? What if people need a certain alignment?

You have a nice function (below) that aligns to a page size; why not just let people use that for page size alignment? 

This is a generic function, not related to boot; might it go in the general purpose package in uroot utils?